orcid-core/src/main/resources/i18n/api_en.properties

```diff
@@ -61,7 +61,7 @@ apiError.9018.userMessage=The ORCID record is locked.
 apiError.9018.moreInfo=http://members.orcid.org/api/api-error-codes
 apiError.9019.userMessage=The put code in the URL did not match that in the body of the message.
 apiError.9019.moreInfo=http://members.orcid.org/api/api-error-codes
-apiError.9020.userMessage=The put code in the URL did not match that in the body of the message.
+apiError.9020.userMessage=Invalid incoming message.
```

Comment:
I would like to see as much detail as possible to be provided to the API user. Custom exceptions with unique 9000 codes sounds good to me.